test/configuration_test.dart

```diff
@@ -153,4 +153,48 @@ Future<void> main([List<String>? args]) async {
     realm1.close();
     realm2.close();
   });
+  
+  test('Configuration - disableFormatUpgrade=true throws error', () async {
+    final realmBundleFile = "test/data/realm_files/realm-bundle.realm";
+    final realmDir = "test/disableFormatUpgradeTrue";
+    final realmPath = combineFilePath(realmDir, Configuration.defaultPath);
+
+    await Directory(realmDir).create();
+    var config = Configuration([Car.schema])..path = realmPath;
+    var realm = Realm(config);
+    realm.close();
+
+    Realm.deleteRealm(realmPath);
+    var file = File(realmBundleFile);
+    await file.copy(realmPath);
```

Comment:
This seems to be unnecessarily complicated. It shouldn't be necessary to open the Realm the first time. Instead, we could do something like:

```dart
final realmBundleFile = "test/data/realm_files/realm-bundle.realm";
final config = Configuration([Car.schema], disableFormatUpgrade: true);
await File(realmBundleFile).copy(config.path);

// try to open the Realm with config
```